lib/src/services/test_data/issue_data.ts

```diff
@@ -23,7 +24,7 @@ export const issue: Issue = {
       subtask: false,
     },
     assignee: {
-      name: "assignee",
+      assigneeName: "assignee",
```

Comment:
The context here (being in the `assignee`) makes the prefix redundant. It would be better if this was just `name` or `displayName`.

Reply:
Since `Author` is using `name`, let's also use `name` for assignee, unless there is a reason not to.

Reply:
Altered so that the `name` field is used instead of the `assigneeName` field